src/easy-popup.js

```diff
@@ -53,7 +53,11 @@
                     triggerSelector: '',
                     hasMobileLayout: false, // has mobile layout, false by default
                     theme: 'default',
+
+                    keyboard: true,
```

Comment:
Please write some words for this new config variable, also update the README file.